docs/syntax-reference.rst

```diff
@@ -166,31 +166,29 @@ timestamp and message key, respectively.
 
 The WITH clause supports the following properties:
 
-========================= ============================================================================================
-Property                  Description
-========================= ============================================================================================
- KAFKA_TOPIC (required)   | The name of the Kafka topic that backs this stream. The topic must already exist in Kafka.
- VALUE_FORMAT (required)  | Specifies the serialization format of the message value in the topic. Supported formats:
-                          | ``JSON``, ``DELIMITED`` (comma-separated value), and ``AVRO``.
- KEY                      | Optimization hint: If the Kafka message key is also present as a field/column in the Kafka
-                          | message value, you may set this property to associate the corresponding field/column with
-                          | the implicit ``ROWKEY`` column (message key).
-                          | If set, KSQL uses it as an optimization hint to determine if repartitioning can be avoided
-                          | when performing aggregations and joins.
-                          | See :ref:`ksql_key_constraints` for more information.
- TIMESTAMP                | By default, the implicit ``ROWTIME`` column is the timestamp of the message in the Kafka
-                          | topic. The TIMESTAMP property can be used to override ``ROWTIME`` with the contents of the
-                          | specified field/column within the Kafka message value (similar to timestamp extractors in
-                          | Kafka's Streams API). Time-based operations such as windowing will process a record
-                          | according to the timestamp in ``ROWTIME``.
-========================= ============================================================================================
-
-
++-------------------------+--------------------------------------------------------------------------------------------+
+| Property                | Description                                                                                |
++=========================+============================================================================================+
```

Comment:
this table does not format properly:
![image](https://user-images.githubusercontent.com/11722533/38328486-ed4d2da8-37ff-11e8-8ae5-29d14b6468d6.png)